.github/workflows/scan-images.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Daily Scan Images
+
+on:
+  schedule:
+    - cron: "0 7 * * *"
+  workflow_dispatch:
+
+jobs:
+  scan-images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: ${{ fromJSON(vars.SCAN_IMAGES_LIST) }}
```

Comment:
Can't we have this for every singe image in the repository instead of using a static list?